api/controllers/search.js

```diff
@@ -454,20 +455,47 @@ var searchCollection = async function (roles, keywords, schemaName, pageNum, pag
     ]
   };
 
+
+  
   console.log("modifier:", modifier);
   console.log("match:", match);
 
-  var sortingValue = {};
-  sortingValue[sortField] = sortDirection;
+
+  if (schemaName == "Document"){
```

Comment:
This is a great start, but I don't think it is going to keep the same sorting behaviour we currently have. By default the documents should sort by the date and display name; however, the user has the ability to click on a header which would indicate they only want to sort by a single column.